app/src/main/java/com/wednesday/bankingdetails/data/FavouritesDatabase.kt

```diff
@@ -0,0 +1,31 @@
+package com.wednesday.bankingdetails.data
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(entities = [FavouriteBankEntity::class], version = 1)
+abstract class FavouritesDatabase : RoomDatabase() {
+
+    abstract fun favouriteBankDao(): FavouriteBankDao
+
+    companion object {
+        private var INSTANCE: FavouritesDatabase? = null
+
+        fun getDatabase(context: Context): FavouritesDatabase? {
```

Comment:
Why does this return an optional? When will this ever be null?

Reply:
No. Changed to non-null